website/src/components/Chat/InferencePoweredBy.tsx

```diff
@@ -0,0 +1,30 @@
+import { Grid, Heading } from "@chakra-ui/react";
+import { TeamMember } from "src/components/TeamMember";
+
+import data from "../../data/team.json";
+
+const SponsorGroup = ({ heading, members }) => {
```

Comment:
Would be great if we could rename this to something like `MemberGroup` and move it to its own file, there is nothing about this that is sponsor specific